site/content/in-dev/unreleased/evolution.md

```diff
@@ -0,0 +1,88 @@
+---
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+title: Polaris Evolution
+type: docs
+weight: 1000
+---
+
+This page discusses what can be expected from Apache Polaris as the project evolves.
+
+## Using Polaris as a Catalog
+
+Polaris is primarily intended to be used as a Catalog of Tables and Views. As such, 
+it implements the Iceberg REST API, Polaris Management API and the Generic Tables API.
+
+Revisions of the Iceberg REST API are controlled by the [Apache Iceberg](https://iceberg.apache.org/)
+community. Polaris accurately implements this specification. However, there
```

Comment:
Sorry, I know that we've gone back and forth on this wording a lot. Can we try `Polaris implements this specification, but there is no guarantee ...` to make this flow a little better?

I feel like `Polaris accurately...` doth protest too much. I actually thought that `attempts to` was fine, since after all we don't actually implement every API right now. We just try to :)

Reply:
rephrases back to "attempts to" and added a note about optional APIs... Is it better? :sweat_smile: